examples/example.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const { createReadStream } = require('fs')
+const { resolve } = require('path')
+const { fastifyThrottle } = require('../index')
+const { RandomStream } = require('../test/utils/random-stream')
+
+async function main () {
+  const fastify = require('fastify')()
+
+  await fastify.register(fastifyThrottle, {
+    bps: 10000,
+    streamPayloads: true,
+    stringPayloads: true,
+    bufferPayloads: true
+  })
+
+  fastify.get('/string', (req, reply) => {
+    reply.send(Buffer.allocUnsafe(1024 * 1024).toString('ascii'))
+  })
+
+  fastify.get('/buffer', (req, reply) => {
+    reply.send(Buffer.allocUnsafe(1024 * 1024))
+  })
+
+  fastify.get('/stream', (req, reply) => {
+    reply.send(new RandomStream(30000))
+  })
+
+  fastify.get('/delayed', {
+    config: {
```

Comment:
Is there a type augmentation in index.d.ts for this? I can't find it

Reply:
Yes?! It is the FastifyContextConfig which is extended.